packages/block-editor/src/components/inserter/menu.native.js

```diff
@@ -182,6 +183,7 @@ function InserterMenu( {
 			onKeyboardHide={ onKeyboardHide }
 			header={
 				<>
+					<NoticeList />
```

Comment:
Just checking to verify that reference of the `<NoticeList />` was required for the `Notice` to be shown. IIRC, the `<NoticeList />`is being rendered in the view hierarchy so all underlying views should be able to utilize it. Let me know your thoughts on this.

Reply:
Yes, indeed. But it's being rendered behind the `BottomSheet`, which is taking the entire screen. Another possibility is to add the `<NoticeList />` in the `BottomSheet` itself, then the `InserterMenu` doesn't need to know about it and is more portable to be used in other parts. What do you think?

Reply:
Actually, maybe we need to add directly to the `BottomSheet`, so it doesn't have that space on top of the `Notice`.

<img src="https://user-images.githubusercontent.com/628412/134308168-3ead9e7f-dd36-4c5a-b104-e411e93b8c2e.gif" width=350 />

Reply:
Ah, thanks for these concerns. As noted in our conversation, let's put all of these scenarios in the PR for more clarity so we can get some more eyes on it 👍🏾